### PR TITLE
Anoxon/remove dup defines

### PR DESCRIPTION
**Pull request by alexa-noxon**
Description
-----------

Checklist:
----------

- [ ] I have tested my changes. No regression in existing tests.
- [ ] My code is Linted.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: demos/dev_mode_key_provisioning/src/aws_dev_mode_key_provisioning.c [hunk 1]
```diff
@@ -764,6 +764,67 @@ CK_RV xProvisionDevice( CK_SESSION_HANDLE xSession,
...
+
+    if( ( xResult == CKR_OK ) || ( xResult == CKR_CRYPTOKI_ALREADY_INITIALIZED ) )
+    {
+        xResult = xGetSlotList( &pxSlotId, &xSlotCount );
+    }
+
+    if( xResult == CKR_OK )
+    {
+        /* Check if token is initialized */
+        pxTokenInfo = pvPortMalloc( sizeof( CK_TOKEN_INFO ) );
```
Comment (dan4thewin) [inline demos/dev_mode_key_provisioning/src/aws_dev_mode_key_provisioning.c]: Do we need to check this for failure?

Comment (alexa-noxon) [reply demos/dev_mode_key_provisioning/src/aws_dev_mode_key_provisioning.c]: Yes good catch updated the code now.

File: demos/dev_mode_key_provisioning/src/aws_dev_mode_key_provisioning.c [hunk 2]
```diff
@@ -764,6 +764,75 @@ CK_RV xProvisionDevice( CK_SESSION_HANDLE xSession,
...
+    }
+
+    if( CKR_OK == xResult )
+    {
+        xTokenFlags = pxTokenInfo->flags;
+    }
+
+    if( ( CKR_OK == xResult ) && !( CKF_TOKEN_INITIALIZED & xTokenFlags ) )
+    {
+        /* If token is not initialized - initialize it */
```
Comment (dcgaws) [inline demos/dev_mode_key_provisioning/src/aws_dev_mode_key_provisioning.c]: "If the token is not initialized, initialize it."

File: libraries/abstractions/pkcs11/test/iot_test_pkcs11.c [hunk 3]
```diff
@@ -1497,9 +1492,9 @@ TEST( Full_PKCS11_EC, AFQP_Sign )
     CK_OBJECT_HANDLE xPublicKeyHandle;
     CK_OBJECT_HANDLE xCertificateHandle;
     /* Note that mbedTLS does not permit a signature on all 0's. */
```
Comment (dcgaws) [inline libraries/abstractions/pkcs11/test/iot_test_pkcs11.c]: I don't think that's accurate. Rather, the issue is in trying to do the DSA computation with a zero exponent.

File: demos/dev_mode_key_provisioning/src/aws_dev_mode_key_provisioning.c (hunk 2 above)
Comment (alexa-noxon) [reply demos/dev_mode_key_provisioning/src/aws_dev_mode_key_provisioning.c]: fixed.

File: libraries/abstractions/pkcs11/test/iot_test_pkcs11.c (hunk 3 above)
Comment (alexa-noxon) [reply libraries/abstractions/pkcs11/test/iot_test_pkcs11.c]: thanks for pointing that out- fixed the comment here and other places.

**Pull request merged**